doc/design/concurrent_data_transfer_and_kernel_execution.md

```diff
@@ -48,21 +47,20 @@ class Channel {
 
 ```
 ...
-chan_list_name = "chan_list"
-with fluid.go(concurrent_program, chan_list_name):
+chan_list_config = fluid.channel_list.config(name="chan_list", type=var.CHANNEL_LIST)
+
+with fluid.go(concurrent_program, chan_list_config) as go:
     image = fluid.layers.data_layer(...)
     label = fluid.layers.data_layer(...)
-    chan_list = fluid.channel_list.make(type=var.CHANNEL_LIST, name=chan_list_name)
     places = get_places() 
     with parallel.for(places) as for_loop:
-        chan = fluid.channel_list.get_channel(chan_list, type=var.CHANNELTYPE,name="chan_{}".format(for_loop.i))
+        chan = fluid.channel_list.get_channel(go.chan_list, type=var.CHANNELTYPE,name="chan_{}".format(for_loop.i))
         fluid.channle.send(chan, data=[for_loop.input(image), for_loop.input(label)])
 
-with fluid.go(main_program, chan_list_name):
-    chan_list = fluid.channel_list.make(type=var.CHANNEL_LIST, name=chan_list_name)
+with fluid.go(main_program, chan_list_config) as go:
     places = get_places() 
     with parallel.for(places) as for_loop:
-        chan = fluid.channel_list.get_channel(chan_list, type=var.CHANNELTYPE,name="chan_{}".format(for_loop.i))
+        chan = fluid.channel_list.get_channel(go.chan_list, type=var.CHANNELTYPE,name="chan_{}".format(for_loop.i))
```

Comment:
Data reading has been done on Python side.
In the case of multi-GPU, since different GPU have the different address, each GPU should own one channel. So I use multiple channel here.